src/tsjson.test.ts

```diff
@@ -306,6 +306,30 @@ describe("More involved tests with objects", () => {
     const toFail = {};
     expect(() => parser.parse(JSON.stringify(toFail))).toThrow();
   });
+
+  test("Object disallowing additionalProperties", () => {
```

Comment:
Recreating [my original comment from the other PR](https://github.com/ostrowr/ts-json-validator/pull/29#discussion_r377218524) here:

Let me know if you want me to add another test where the schema defined `additionalProperties: false`, but the `removeAdditional` option is omitted. I thought that might be overkill since that's more towards testing ajv's internals, which we may not need to worry about here.